src/http/tx-scheduler/Transactions.test.ts

```diff
@@ -0,0 +1,296 @@
+import { HttpClient } from '../client/http'
+import {
+  IDeployContractRequest,
+  IETHTransactionParams,
+  IHttpGETRequest,
+  IHttpPOSTRequest,
+  IJobResponse,
+  ILog,
+  IScheduleResponse,
+  ISearchRequest,
+  ISendRawRequest,
+  ISendTransactionRequest,
+  ISendTransferRequest,
+  ITransactionResponse
+} from '../types'
+
+import { TxSchedulerTransactions } from './Transactions'
+
+const mockJobLogRes: ILog = {
+  message: 'Job Created',
+  status: 'CREATED',
+  createdAt: new Date()
+}
+
+const mockJobRes: IJobResponse = {
+  uuid: 'jobUUID',
+  chainUUID: 'chainUUID',
+  logs: [mockJobLogRes],
+  transaction: {},
+  status: 'STARTED',
+  type: 'eth://ethereum/transaction',
+  createdAt: new Date()
+}
+
+const mockScheduleRes: IScheduleResponse = {
```

Comment:
please use the convention `fakeX` for fake data and `mockX` for mocked classes and packages. Also no need to type your fake objects as you will need to fill every single field when you most of the time don't care about all the fields.